webextensions/api/browserAction.json

```
@@ -40,7 +40,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/ColorArray",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "20"
```

:x: https://github.com/chromium/chromium/commit/e28a937b1cb87e62afefb5918512ce8b9dfc49d5 landed in [19.0.1075.0](https://github.com/chromium/chromium/releases/tag/19.0.1075.0) before 19.0.1084.

```suggestion
                "version_added": "19"
```